app/models/profile.rb

```diff
@@ -90,4 +90,9 @@ def encrypt_compound_pii_fingerprint(pii)
     compound_pii_fingerprint = self.class.build_compound_pii_fingerprint(pii)
     self.name_zip_birth_year_signature = compound_pii_fingerprint if compound_pii_fingerprint
   end
+
+  def liveness_check?
```

Comment:
Maybe `has_liveness_check?` or `had_liveness_check?` or `includes_liveness_check?` to clarify that the liveness check already happened?

Reply:
The linter complained about has_